docs/HowTo/topics/Topic-overview.md

```diff
@@ -0,0 +1,50 @@
+# Topic Overview
+
+Topic overview provides different views of topic. 
+
+## Overview
+
+In this tab, the below are displayed
+
+- topic configuration like partitions, replication factor and if any advanced configuration
+- cluster details like bootstrap url
+- count of producer and consumer subscriptions and schemas
+
+
+## Subscriptions
+
+In this tab, all producer and consumer acls are displayed, including prefixed and transactional.
```

Comment:
Retrieve passwords here for consumer and producer acls that belong to your team.